### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by benoitgrelard**

### CODE REVIEW

**Comment by chaance** [comment]
Is this not going to clash with @jjenzz's work in https://github.com/radix-ui/website/pull/51?

**Comment by benoitgrelard** [comment]
No she’s going from the top and I’m going from the bottom and we’re chatting about it 👍

File: pages/primitives/docs/utilities/portal.mdx [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default () => (
       type: 'React.RefObject<HTMLElement>',
       typeSimple: 'ref',
       description:
-        'A optional ref to a different container where the portal should be appended.',
+        'A optional ref to a different container where the portaled content should be appended.',
```
Comment (jjenzz) [inline pages/primitives/docs/utilities/portal.mdx:52]: Should this be `portalled` with two l's ? If not, perhaps we need to rename our `portalled` prop?

File: pages/primitives/docs/utilities/label.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Label
 name: label
 version: 0.0.1
-description: A primitive useful for rendering labels.
+description: A component use to render accessible label associated with controls.
```
[the remark this answers is not in the document]
Comment (jjenzz) [reply pages/primitives/docs/utilities/label.mdx]: This one is outdated now so can maybe leave as is. I'll leave it up to you!

**Pull request merged**